### PR TITLE
4-dir sprites for radio intercoms.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Adds in directional for radio intercoms, shifts the other wallframes sprites by 1/2 pixels south.

## Why It's Good For The Game
More directionals less dirty var editing.

## Changelog
:cl:
imageadd: Added in directional sprites for intercoms.
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
Mapping freeze until some shuttle stuff is merged

**Comment by Ghommie** [comment]
so yea, hold on I'll have to remap them.

**Comment by Ghommie** [comment]
Should be ok now.

**Comment by Rohesie** [comment]
Might conflict with @KwinOriginal 's work.

File: _maps/map_files/BigRed_v2/BigRed_v2.dmm [hunk 1]
```diff
@@ -731,6 +731,7 @@
 	},
 /obj/item/radio/intercom{
 	dir = 1;
+	freqlock = 1;
```
Comment (DominikPanic) [inline _maps/map_files/BigRed_v2/BigRed_v2.dmm]: No

File: _maps/map_files/BigRed_v2/BigRed_v2.dmm [hunk 2]
```diff
@@ -4019,6 +4020,7 @@
 /area/bigredv2/caves/lambda_lab)
 "alZ" = (
 /obj/item/radio/intercom{
+	freqlock = 1;
```
Comment (DominikPanic) [inline _maps/map_files/BigRed_v2/BigRed_v2.dmm]: Dirty

**Comment by Rohesie** [comment]
New conflict. Let us know when it's solved so we can merge it.

**Comment by Ghommie** [comment]
It should be ready now.

**Comment by Rohesie** [comment]
Let us know when you fix the merge conflicts so we can give it priority for testing.

**Pull request merged**